motionplan/plannerOptions.go

```diff
@@ -217,20 +235,47 @@ func (p *plannerOptions) SetMinScore(minScore float64) {
 
 // addPbConstraints will add all constraints from the protobuf constraint specification. This will deal with only the topological
 // constraints. It will return a bool indicating whether there are any to add.
-func (p *plannerOptions) addPbTopoConstraints(from, to spatialmath.Pose, constraints *Constraints) bool {
+func (p *plannerOptions) addPbTopoConstraints(
```

Comment:
This function references pb Constraints, which is no longer the case we have our own notion of Constraints that we construct from it.  We should remove the reference to this in the comment and function name.  

I think if we do this, and recontextualize the `Constraints` object as "our version of constraints which go above and beyond the nominal set defined by proto" then I am on board with adding Psudeolinear to the constructor per our other discussion

Reply:
Updated to remove all references to proto in the function name and description.

I left in the couple of comments in the body of the function referencing proto, since those are specific TODOs and the proto is part of the explanation of why/what work needs to be done.